### DISCUSSION TITLE
[Script request] ProjectSend

### DISCUSSION THREAD

**Discussion by nodeam** [community-scripts/ProxmoxVE#346] [Request script]
### Application Name

ProjectSend

### Website

https://github.com/projectsend/projectsend

### Description

ProjectSend is a free, clients-oriented, private file sharing web application.

Clients are created and assigned a username and a password.
Uploaded files can be assigned to specific clients or clients groups.

Other features include auto-expiration of upload, notifications, full logging of actions by users and clients, option to allow clients to also upload files, themes, multiple languages

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by bvdberg01** [comment]
@MickLesk I can look into this one.

**Reply by MickLesk** [reply]
accept invite - after this you can move it from project board to in progress 👍🏻

**Comment by batoorr** [comment]
Hello, Thank You for this script, i however have a problem, the installation goes thru without any errors, I am able to use app, however it is EXTREMLY SLOW (the dashboard). After i log in, clicking any option it takes around 30-60s for the app to respond. The proxmox summary shows the projectsend is using around 0.3% of assigned CPU at best. I'm running r1720, installed by the script provided on this website. My home lab resources aren't  the issue.  Any help would be appriciated.

**Reply by michelroegl-brunner** [reply]
Best ask the devs of the App? We only provide a way to install and no app support

**Reply by batoorr** [reply]
You are probably right, was just wondering if it might be a issue with how the installation was done.
